sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_RecognizeHealthcareEntities.md

```diff
@@ -0,0 +1,210 @@
+            Console.WriteLine($"    Length: {entity.Length}");
+            Console.WriteLine($"    IsNegated: {entity.IsNegated}");
+            Console.WriteLine($"    Links:");
+
+            foreach (HealthcareEntityLink healthcareEntityLink in entity.Links)
+            {
+                Console.WriteLine($"        ID: {healthcareEntityLink.Id}");
+                Console.WriteLine($"        DataSource: {healthcareEntityLink.DataSource}");
+            }
+        }
+
+        Console.WriteLine($"    Document statistics:");
+        Console.WriteLine($"        Character count (in Unicode graphemes): {result.Statistics.Value.CharacterCount}");
+        Console.WriteLine($"        Transaction count: {result.Statistics.Value.TransactionCount}");
+        Console.WriteLine("");
+    }
+    Console.WriteLine($"Request statistics:");
+    Console.WriteLine($"    Document Count: {results.Statistics.DocumentCount}");
+    Console.WriteLine($"    Valid Document Count: {results.Statistics.ValidDocumentCount}");
+    Console.WriteLine($"    Transaction Count: {results.Statistics.TransactionCount}");
+    Console.WriteLine($"    Invalid Document Count: {results.Statistics.InvalidDocumentCount}");
+    Console.WriteLine("");
+}
+```
+
+To recognize healthcare entities in a collection of documents in different languages, call `StartHealthcareBatch` on an `IEnumerable` of `TextDocumentInput` objects, setting the `Language` on each document.
+
+```C# Snippet:TextAnalyticsSampleHealthcareBatch
+    string document = @"RECORD #333582770390100 | MH | 85986313 | | 054351 | 2/14/2001 12:00:00 AM | CORONARY ARTERY DISEASE | Signed | DIS | \
+                        Admission Date: 5/22/2001 Report Status: Signed Discharge Date: 4/24/2001 ADMISSION DIAGNOSIS: CORONARY ARTERY DISEASE. \
+                        HISTORY OF PRESENT ILLNESS: The patient is a 54-year-old gentleman with a history of progressive angina over the past several months. \
+                        The patient had a cardiac catheterization in July of this year revealing total occlusion of the RCA and 50% left main disease ,\
+                        with a strong family history of coronary artery disease with a brother dying at the age of 52 from a myocardial infarction and \
+                        another brother who is status post coronary artery bypass grafting. The patient had a stress echocardiogram done on July , 2001 , \
+                        which showed no wall motion abnormalities , but this was a difficult study due to body habitus. The patient went for six minutes with \
+                        minimal ST depressions in the anterior lateral leads , thought due to fatigue and wrist pain , his anginal equivalent. Due to the patient's \
+                        increased symptoms and family history and history left main disease with total occasional of his RCA was referred for revascularization with open heart surgery.";
+
+    List<string> batchInput = new List<string>()
```

Comment:
this sample should use  `IEnumerable` of `TextDocumentInput`` like the text suggests

Reply:
updated to string for simplicity